src/api/wallet/WalletApi.ts

```diff
@@ -21,7 +21,7 @@ import {
   Subscriptions,
 } from '@gnosis.pm/dapp-ui'
 
-import { logDebug, toBN } from 'utils'
+import { logDebug, toBN, encoder } from 'utils'
```

Comment:
I think the name `encoder` is too general. Can you rename to something more with in the direction of `encodeUserEarmark` or similar?

Reply:
Bu it is general
It's just an encoder that can encode any flags into a string. We just use it for gasPrice
I'll do `import {encoder as encodeUserEarmark}`, but it should stay encoder at the point of export

Reply:
Yes, it's general, but by the name of `encoder` no-one would ever guess what is what it does.

Reply:
_no-one would ever guess_, I like that <sub>joke</sub>

:+1: